### PR TITLE
Extended "Assisting with Translations" document

### PR DESCRIPTION
**Pull request by Komodorpudel**
This PR adds the sections "Getting started with the Translation Team" and "Responsibilities and Tasks for each User Role" to the "Assisting with Translations" document, while the original content of the doc is slightly changed and now serves as section "Handling Translations on GitHub".

@wbnns there you go :)

### CODE REVIEW

File: docs/assisting-with-translations.md [hunk 1]
```diff
@@ -1,20 +1,17 @@
...
 
-### How To Translate
+This document provides information regarding translations for bitcoin.org.
 
-You can join a translation team on
-[Transifex](https://www.transifex.com/projects/p/bitcoinorg/) and start
-translating or improving existing translations.
+## Table of Contents
+* [Handling Translations on GitHub](#handling-translations-on-github)
+* [Getting started with the Translation Team](#getting-started-with-the-translation-team)
```
Comment (wbnns) [inline docs/assisting-with-translations.md]: Please capitalize "started" so it matches the capitalization style found in the other headings:

"Getting Started with the Translation Team"

File: docs/assisting-with-translations.md [hunk 2]
```diff
@@ -68,3 +65,90 @@ translations at once:
     
     # Push changes back to Transifex
     tx push -s -t -f --skip --no-interactive
+    
+___
+
+## Getting started with the Translation Team
```
Comment (wbnns) [inline docs/assisting-with-translations.md]: Please capitalize "Started" (see comment above).

File: docs/assisting-with-translations.md [hunk 3]
```diff
@@ -1,20 +1,17 @@
...
-  publication.
-* Once reviewed, translations can be [submitted](#import-translations) in a pull
-  request on GitHub.
-* **In doubt, please contact coordinators on Transifex. That'll be much
-  appreciated.**
+___
+
+## Handling Translations on GitHub
+
+This section outlines how to handle translations on GitHub. If you only want to help by translating content, scroll down to section [Getting started with the Translation Team](#getting-started-with-the-translation-team).
```
Comment (wbnns) [inline docs/assisting-with-translations.md]: Please change:

scroll down to section

to:

scroll down to the section,

File: docs/assisting-with-translations.md [hunk 4]
```diff
@@ -68,3 +65,90 @@ translations at once:
...
+
+## Getting started with the Translation Team
+
+While the website itself is managed on [GitHub](https://github.com/bitcoin-dot-org/bitcoin.org/), the translations are handled through [Transifex](https://www.transifex.com/bitcoinorg/bitcoinorg/).
+This section shall explain the first steps on Transifex for new volunteers who want to help with translating bitcoin.org.
+
+General guidelines provided by Transifex itself can be found [here](https://docs.transifex.com/getting-started/translators).
+
+### 1. Create a Transifex Account
+* Follow this [link](https://www.transifex.com/signup/) and create an account. Not much information is needed.
```
Comment (wbnns) [inline docs/assisting-with-translations.md]: Please add a sentence that creating an account on Transifex is free.

File: docs/assisting-with-translations.md [hunk 5]
```diff
@@ -68,3 +65,90 @@ translations at once:
...
+
+### 2. Login and join the Bitcoin.org Translation Team
+
+* Follow this [link](https://www.transifex.com/bitcoinorg/bitcoinorg/) and click on “Join team”.
+* Select the language you want to translate into.
+* Your request to join a team will be accepted instantly, and you will be a translator for the language you selected.
+* If your language is not available yet, close the pop-up, scroll down, and click on “Request language” on the right side. Please consider first if it is necessary that bitcoin.org is available in your language. There is a lot of content to be translated and especially for languages with a low number of native speakers, there won’t be many people available that are willing to assist you.
+
+### 3.	Get to know the Interface
```
Comment (wbnns) [inline docs/assisting-with-translations.md]: Please fix spacing.

File: docs/assisting-with-translations.md [hunk 6]
```diff
@@ -68,3 +65,90 @@ translations at once:
...
+* Play around with the interface. Transifex's interface can be a bit confusing and it cannot hurt to take a look around.
+* As Translator, you cannot cause any harm as you can only edit unreviewed strings. A complete history is saved for every string, making it impossible to destroy previous work.
+* For the beginning, stay away from the Glossary as this can be edited by new translators but no history is saved.
+
+### 4.	Join our Telegram Group
```
Comment (wbnns) [inline docs/assisting-with-translations.md]: Please fix spacing.

File: docs/assisting-with-translations.md [hunk 7]
```diff
@@ -68,3 +65,90 @@ translations at once:
...
+* Join our [Telegram Group](https://t.me/joinchat/Bgh47RC1BZb2YE6u8iznOg).
+* The website maintainer, both Team Leaders for translations, a number of language coordinators, and various translators are present in this group.
+* We are happy to help in case you need assistance.
+
+### 5.	Choose what you want to translate
```
Comment (wbnns) [inline docs/assisting-with-translations.md]: Please fix spacing.

File: docs/assisting-with-translations.md [hunk 8]
```diff
@@ -68,3 +65,90 @@ translations at once:
...
+* Click on "Dashboard" on the top.
+* Once you are on the "Dashboard", click on "Languages" on the left side and select your language.
+* You will see a lot of different resources and their progress.
+* Each resource consists of a number of strings, with a string being for example a paragraph or headline.
+* Each string has three possible states: "untranslated", "translated but unreviewed", and "reviewed". Only the first state "untranslated" is relevant for new volunteers. However, if you find a "translated but unreviewed" string that contains obvious mistakes, you are free to correct them. "Reviewed" strings can only be changed or unreviewed by reviewers.
+* **The first resource "bitcoin.org" contains all strings of the main page. If you are no Bitcoin expert, start here.**
+* **Everything else that follows starts with "devdocs...", indicating that these files are part of the developer documentation. It is recommended that you only try to translate the developer documentation if you are an experienced Bitcoin user and/or developer with a profound understanding.**
+
+### 6.	Start translating
```
Comment (wbnns) [inline docs/assisting-with-translations.md]: Please fix spacing.

File: docs/assisting-with-translations.md [hunk 9]
```diff
@@ -68,3 +65,90 @@ translations at once:
...
+* You must be a native speaker for the language you choose to translate.
+* Please be careful to preserve the original meaning of each text.
+* Sentences and popular expressions should sound native in your language.
+* Translations need to be reviewed by a reviewer or coordinator before
+  publication.
+* Once reviewed, coordinators will notify the Team Leaders that a certain translation is ready for publication.
+* **In doubt, please contact coordinators on Transifex. That'll be much
+  appreciated.**
+  
+### 7.	Take a look at the Responsibilities and Tasks for each User Role
```
Comment (wbnns) [inline docs/assisting-with-translations.md]: Please fix spacing.

File: docs/assisting-with-translations.md [hunk 10]
```diff
@@ -68,3 +65,90 @@ translations at once:
...
+### 1. Team Leaders
+Team Leaders are currently Simon AKA “Komodorpudel” (Telegram: “Komodorpudel”) and Apple AKA “ajsolbrilla”.
+
+* Oversight on the complete translation efforts on Transifex.
+* Keeping track on everything.
+* Being a contact person for all sorts of questions that cannot be answered by language coordinators.
+* Promoting or demoting users (e.g. promoting a reviewer to coordinator).
+* Managing groups that have no active coordinator.
+
+### 2.	Coordinators
```
Comment (wbnns) [inline docs/assisting-with-translations.md]: Please fix spacing.

File: docs/assisting-with-translations.md [hunk 11]
```diff
@@ -68,3 +65,90 @@ translations at once:
...
+
+* Follow this [link](https://www.transifex.com/bitcoinorg/bitcoinorg/) and click on “Join team”.
+* Select the language you want to translate into.
+* Your request to join a team will be accepted instantly, and you will be a translator for the language you selected.
+* If your language is not available yet, close the pop-up, scroll down, and click on “Request language” on the right side. Please consider first if it is necessary that bitcoin.org is available in your language. There is a lot of content to be translated and especially for languages with a low number of native speakers, there won’t be many people available that are willing to assist you.
+
+### 3.	Get to know the Interface
+* Play around with the interface. Transifex's interface can be a bit confusing and it cannot hurt to take a look around.
+* As Translator, you cannot cause any harm as you can only edit unreviewed strings. A complete history is saved for every string, making it impossible to destroy previous work.
+* For the beginning, stay away from the Glossary as this can be edited by new translators but no history is saved.
```
Comment (wbnns) [inline docs/assisting-with-translations.md]: Please change:

For the beginning,

to:

In the beginning,

File: docs/assisting-with-translations.md [hunk 12]
```diff
@@ -68,3 +65,90 @@ translations at once:
...
+
+* Oversight on the complete translation efforts on Transifex.
+* Keeping track on everything.
+* Being a contact person for all sorts of questions that cannot be answered by language coordinators.
+* Promoting or demoting users (e.g. promoting a reviewer to coordinator).
+* Managing groups that have no active coordinator.
+
+### 2.	Coordinators
+Various people across all language teams are coordinators. For a number of languages, no active coordinator exists. If there are any questions or you want to assist by becoming a coordinator, write one of the Team Leaders.
+* If man power is needed: Translating, thereby using translations provided by the glossary if applicable and striving for consistency across strings.
```
Comment (wbnns) [inline docs/assisting-with-translations.md]: Please change:

If man power is needed:

to:

When additional help is needed:

File: docs/assisting-with-translations.md [hunk 13]
```diff
@@ -68,3 +65,90 @@ translations at once:
...
+* Being a contact person for all reviewers and translators within a specific language team.
+* Introducing and helping new volunteers.
+* Promoting or demoting users (e.g. promoting a translator to reviewer).
+* Kicking out users that do not behave (e.g. only use Google Translator) or are completely inactive.
+
+### 3. Reviewers
+* Translating, thereby using translations provided by the glossary if applicable and striving for consistency across strings.
+* Reviewing strings (preferably not your own strings if possible).
+* Checking translations for correctness regarding meaning and spelling.
+* Checking for consistency across translations (e.g. is “transaction malleability” translated consistently across all string?).
```
Comment (wbnns) [inline docs/assisting-with-translations.md]: "string" should be "strings" (plural)

File: docs/assisting-with-translations.md [hunk 14]
```diff
@@ -68,3 +65,90 @@ translations at once:
...
+
+### 4. Translators
+* Translating, thereby using translations provided by the glossary if applicable and striving for consistency across strings.
+* Extending the glossary with translations for necessary and general term
```
Comment (wbnns) [inline docs/assisting-with-translations.md]: "term" should be "terms" (plural)

File: docs/assisting-with-translations.md (hunk 1 above)
Comment (Komodorpudel) [reply docs/assisting-with-translations.md]: Fixed

File: docs/assisting-with-translations.md (hunk 5 above)
Comment (Komodorpudel) [reply docs/assisting-with-translations.md]: I can not observe this issue. Also the preview file looks OK. Can you please check?

Comment (Komodorpudel) [reply docs/assisting-with-translations.md]: Alright, could observe it a few minutes ago. Removed space and added it again. Should be fine now.

**Comment by wbnns** [comment]
@Komodorpudel Thanks for the fixes. Looks great.

**Pull request merged**